src/commands/utils/detectors.ts

```diff
@@ -1,60 +1,66 @@
 import { Errorable, combine, failed, getErrorMessage } from "./errorable";
-import AksClusterTreeItem from '../../tree/aksClusterTreeItem';
-import * as fs from 'fs';
-import * as path from 'path';
-import { ARMResponse, CategoryDetectorARMResponse, SingleDetectorARMResponse, isCategoryDataset } from "../../webview-contract/webviewDefinitions/detector";
+import AksClusterTreeItem from "../../tree/aksClusterTreeItem";
+import * as fs from "fs";
+import * as path from "path";
+import {
+    ARMResponse,
+    CategoryDetectorARMResponse,
+    SingleDetectorARMResponse,
+    isCategoryDataset,
+} from "../../webview-contract/webviewDefinitions/detector";
 import { getResourceManagementClient } from "./clusters";
-const tmp = require('tmp');
-const meta = require('../../../package.json');
+import { dirSync } from "tmp";
```

Comment:
Not sure why we were using `require()` for importing `tmp` functions, so I thought I'd better call this out in case there's a reason I don't understand. It seems to build and run fine, so I _think_ this change is fine.